modules/graphql_content/src/Plugin/GraphQL/Fields/EntityById.php

```diff
@@ -19,12 +19,14 @@
  *   multi = false,
  *   weight = -1,
  *   arguments = {
- *     "id" = "String"
+ *     "id" = "Int",
```

Comment:
Actually String is correct here. Interestingly, the entity system does not enforce integer ids. What you could do, however, would be to move the argument into the deriver. So that we can check the type of the entity id key on a per-entity-type basis.